<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>731</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <widget class="QWidget" name="Home_Tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Factibilidad_Tab">
     <attribute name="title">
      <string>Factibilidad</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Battery_Tab">
     <attribute name="title">
      <string>Battery status</string>
     </attribute>
     <widget class="QPushButton" name="Button1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>551</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Button1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Camera_Tab">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="SerialCom_Tab">
     <attribute name="title">
      <string>Serial Communication</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>261</width>
        <height>68</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="verticalSpacing">
        <number>22</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="SerialPort_ComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BaudRate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="BaudRate_ComboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="OpenButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CloseButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RefreshButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="About_Tab">
     <attribute name="title">
      <string>About</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
